core/templates/dev/head/domain/topic/TopicUpdateService.js

```diff
@@ -276,8 +335,8 @@ oppia.factory('TopicUpdateService', [
        * uncategorized skill ids and records the change in
        * the undo/redo service.
        */
-      moveSkillIdToSubtopic: function(
-          topic, oldSubtopicId, newSubtopicId, skillId) {
+      moveSkillToSubtopic: function(
+          topic, oldSubtopicId, newSubtopicId, skillId, skillDescription) {
```

Comment:
Here and below, it would probably be better to combine skillId and skillDescription into one "thing" (see above).

Reply:
Done, changed it to a skill dict.